etna/transforms/outliers/point_outliers.py

```diff
@@ -120,14 +140,28 @@ def detect_outliers(self, ts: TSDataset) -> Dict[str, List[pd.Timestamp]]:
             distance_func=self.distance_func,
         )
 
+    def params_to_tune(self) -> Dict[str, "BaseDistribution"]:
+        """Get default grid for tuning hyperparameters.
+
+        Returns
+        -------
+        :
+            Grid to tune.
+        """
+        return {
+            "window_size": IntUniformDistribution(low=3, high=30),
+            "distance_coef": UniformDistribution(low=0.5, high=5),
+            "n_neighbors": IntUniformDistribution(low=1, high=10),
+        }
+
 
 class PredictionIntervalOutliersTransform(OutliersTransform):
     """Transform that uses :py:func:`~etna.analysis.outliers.prediction_interval_outliers.get_anomalies_prediction_interval` to find anomalies in data."""
 
     def __init__(
         self,
         in_column: str,
-        model: Union[Type["ProphetModel"], Type["SARIMAXModel"]],
+        model: Union[Literal["prophet"], Literal["sarimax"], Type["ProphetModel"], Type["SARIMAXModel"]],
```

Comment:
It looks somewhat like a hack. I don't think it is a good solution.

Reply:
I is because categorical distribution can't work with this type?

Reply:
Yes, [documentation with explanation](https://optuna.readthedocs.io/en/stable/reference/generated/optuna.distributions.CategoricalDistribution.html).